app/assets/stylesheets/common/admin/admin_base.scss

```diff
@@ -12,6 +12,47 @@ $mobile-breakpoint: 700px;
   --space-6: calc(0.25rem * 6);
 }
 
+.admin-site-settings {
+  &__login,
+  &__notifications,
+  &__trust,
+  &__legal,
+  &__font,
+  &__branding,
+  &__navigation,
+  &__security,
+  &__developer,
+  &__rate_limits,
+  &__experimental {
+    .sidebar-section-link.active[data-link-name="admin_all_site_settings"] {
+      background-color: unset;
```

Comment:
The icon color is different for active sidebar items too:

![image](https://github.com/user-attachments/assets/31037bf7-8aa9-4ea1-a9d0-c352c086ae6d)